lib/Dialect/FIRRTL/Transforms/GrandCentralTaps.cpp

```diff
@@ -847,36 +852,18 @@ void GrandCentralTapsPass::runOnOperation() {
         // Concatenate the prefix into a proper full hierarchical name.
         addSymbol(
             FlatSymbolRefAttr::get(SymbolTable::getSymbolName(rootModule)));
-        if (port.nla && shortestPrefix->empty() &&
-            port.nla.root() != rootModule.moduleNameAttr()) {
-          // This handles the case when nla is not considered for common prefix
-          // stripping.
-          auto rootMod = port.nla.root();
-          for (auto p : path) {
-            if (rootMod == p->getParentOfType<FModuleOp>().getNameAttr())
-              break;
-            addSymbol(getInnerRefTo(p));
-          }
-        }
```

Comment:
This block of code was surprisingly not covered by existing tests after https://github.com/llvm/circt/commit/43035e72f55a35721c1dc9ea66799be3114d93b2. The logic that merged NLA path with the computed instance paths made this unreachable (for the tests we had).